lib/mjsonwp/routes.js

```diff
@@ -23,14 +23,14 @@ const METHOD_MAP = {
   '/wd/hub/session/:sessionId/timeouts': {
     GET: {command: 'getTimeouts'}, // W3C route
     POST: {command: 'timeoutsW3C', payloadParams: {
-      validate: (jsonObj) => {
-        // W3C requires some of script/pageLoad/implicit like: { "script": 1000, "pageLoad": 7000, "implicit": 5000 }
-        // MJSONWP requires both type and ms like: {"type": "command", "ms": 3000}
-        if (!jsonObj.script && !jsonObj.pageLoad && !jsonObj.implicit) {
-          if (!jsonObj.type && !jsonObj.ms) {
-            return 'We require type and ms for MJSONWP or (script|pageLoad|implicit) for W3C';
-          } else if (!jsonObj.type || !jsonObj.ms) {
-            return 'We require type and ms for MJSONWP';
+      validate: (jsonObj, isW3CProtocol) => {
+        if (isW3CProtocol) {
+          if (!jsonObj.script && !jsonObj.pageLoad && !jsonObj.implicit) {
```

Comment:
I'd rather use util.hasValue, since zero values are valid there, but they will be discarded by the current verification